packages/sdk/src/oracle/client.ts

```diff
@@ -23,6 +23,9 @@ export class Client {
     const requester = ethers.utils.getAddress(params.requester);
     return this.sm.types.setActiveRequest.create({ ...params, requester });
   }
+  setActiveRequestByTransaction(params: setActiveRequestByTransaction.Params): string {
+    return this.sm.types.setActiveRequestByTransaction.create(params);
+  }
```

Comment:
this is the new function to call if you want to lookup by transaction from the UI